packages/toolkit/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -601,6 +611,11 @@
             "description": "Called when Step Functions Workflow Studio is closed",
             "metadata": [{ "type": "didSave" }]
         },
+        {
+            "name": "appcomposer_initializeProject",
```

Comment:
As a followup, do you plan to move these local definitions into common? https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json

Reply:
Ah, I wasn't aware of those. I'd need to ask the team if that's something we want to do